packages/sirius-web/backend/sirius-web-table/src/main/java/org/eclipse/sirius/web/table/configuration/SiriusWebTableStarterConfiguration.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright (c) 2024 CEA LIST.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.table.configuration;
+
+import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.context.annotation.ComponentScan;
+
+/**
+ * Auto configuration to discover Table components.
+ *
+ * @author mcharfadi
+ */
+@AutoConfiguration
+@ComponentScan(basePackages = {
+    "org.eclipse.sirius.web.table"
+})
+public class SiriusWebTableStarterConfiguration {
```

Comment:
This should require the META-INF spring configuration file otherwise it shouldn't work. See in `src/main/resources` of `sirius-web-starter`